lib/index.js

```diff
@@ -4,7 +4,9 @@ var web = require('node-static');
 var chalk = require('chalk');
 var fs = require('fs');
 var path = require('path');
+var HTTPStatus = require('http-status');
 
+var REDIRECT_CODE = 301;
```

Comment:
With http-status, you no longer need this, as you have HTTPStatus.MOVED_PERMANENTLY.